map-of-the-web/motw/motw/producing.py

```diff
@@ -0,0 +1,132 @@
+from bokeh.plotting import figure, output_file, show
```

Comment:
An overall comment on this file:  It looks like this is doing the majority of the work for this fist sample interface.  I appreciate that we want something quick and dirty for a first test.

That said, before we go too deep into adding complexity to this application, I would like to be sure we are leveraging the power that Django has to offer!  By using Django we have a framework for organizing the code, and it's going to be important to mirror expectations based on Django's documented best practices.

I see that this module is loaded elsewhere, but it is unclear to me whether this is a utility application or something else.  This is why file names and directory organization are so important.  Django has a [documented directory structure](http://django-project-skeleton.readthedocs.io/en/latest/structure.html) which you are probably familiar with, but worth looking at again